addons/l10n_co/views/res_company.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record id="view_company_uvt_form" model="ir.ui.view">
+            <field name="name">company.uvt.form</field>
+            <field name="model">res.company</field>
+            <field name="inherit_id" ref="account.view_company_inherit_form"/>
+            <field name="arch" type="xml">
+                <field name="accounts_code_digits" position="after">
+                    <field name="uvt" groups="base.group_no_one"/>
```

Comment:
Ok, the thing is it's technical because it's used for python code calculations in the taxes, but I understand what you mean. Would you still put a group on the field? Accountant ? Or nothing?